android-components/components/browser/menu/src/main/res/layout/mozac_browser_menu_item_image_text_checkbox_button.xml

```diff
@@ -50,11 +52,12 @@
         style="@style/Mozac.Browser.Menu.Item.Text"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        android:paddingHorizontal="12dp"
         android:button="@null"
         android:drawablePadding="7dp"
         android:textAlignment="gravity"
         android:gravity="center_vertical"
+        android:paddingEnd="@dimen/mozac_browser_menu_item_container_padding_end"
+        android:paddingStart="8dp"
```

Comment:
splitting 16dp padding in half, so buttons have equal tapping area in the worst case with very long strings in both buttons.